cmd/fyne/internal/templates/data/hello_world_go

```diff
@@ -0,0 +1,14 @@
+package main
+
+import (
+	"fyne.io/fyne/v2/app"
+	"fyne.io/fyne/v2/widget"
+)
+
+func main() {
+	a := app.NewWithID({{ printf "%q" .Details.ID }})
+	w := a.NewWindow("Hello World")
```

Comment:
A minor thing, but should we put the app name here instead?

Reply:
Good idea, thanks!